js/index.js

```diff
@@ -10,6 +10,7 @@ const highEls = document.getElementsByClassName("high")
 const lowEls = document.getElementsByClassName("low")
 const volumeEls = document.getElementsByClassName("volume")
 const avgVolumeEls = document.getElementsByClassName("avgVolume")
+const marketCaps = document.getElementsByClassName("mktCap")
```

Comment:
Small nitpick, marketCapEls is a more appropriate name since the value represents HTML elements.

Reply:
gotcha